day6/README.md

```diff
@@ -0,0 +1,27 @@
+Question of the day:
+
+Implement BFS and DFS, both iteratively and recurively if possible.
+Use them to find a node with a value of `val` in a binary tree, if
+the node exists. If not, just return nothing.
+
+## Ideas
+
+BFS can be done iteratively by queuing up children nodes and exploring
+them in the order they are dequeued. The search cannot be done
```

Comment:
```
for node in unvisited:
  visit(node)

def visit(node):
  unvisited += node.children
```
i think this would be a recursive BFS (with use of a helper function)